float32 startRange
float32 startAngle
float32 endRange
float32 endAngle

float32 gapWidth

#startRange- distance at the point of begining of gap
#startAngle - angle at the point of beginning of gap
#endRange- distance at the ending point of gap
#endAngle - angle at the ending point of gap
#gapWidth - width of gap in terms of euclidean distance between start and end points of gap
